src/core/IPosisBundle.d.ts

```diff
@@ -0,0 +1,6 @@
+// Bundle for programs that are logically grouped
+interface IPosisBundle {
+	// name your processes' image names with initials preceding, like ANI/MyCoolPosisProgram (but the actual class name can be whatever you want)
+	// if your bundle consists of several programs you can pretend that we have a VFS: "ANI/MyBundle/BundledProgram1"
+	install(registerPosisProcess: (imageName: string, constructor: new () => IPosisProcess) => boolean): void;
```

Comment:
That sounds fine to me, that would also fit nicely with my existing `ProcessRegistry` class. maybe name it `IPosisProcessRegistry` so we can potentially define other registries later without worry about confusion? Also, maybe rename the method itself to `register(...)`? Since by being in a `IPosisProcessRegistry` already namespaces it as a POSIS item and it should be obvious that its a process passed in.